modules/storage-filesystem/snapshot_test.go

```diff
@@ -140,6 +143,47 @@ func TestSnapshotStorage_StoreSnapshot(t *testing.T) {
 	})
 }
 
+func Test_FileStorage_MetaStatus(t *testing.T) {
```

Comment:
I would make all of the tests in one file to start with the same prefix for example with `TestSnapshotStorage_` or `Test_FileStorage_` so that if someone would like to run all of the tests in that package they could run it with such a command then `go tests -run ^TestSnapshotStorage ./package`